### ISSUE TITLE
Incorrect warning/error message

### ISSUE THREAD

**Issue by arnaldop** [karatelabs/intellij-plugin#62]
In the below snippet, the line `* if (hideLogRegister) utils.setLogLevel('ERROR')` is getting a missing semicolon warning.
Related to https://github.com/karatelabs/intellij-plugin/issues/59.

```
@ignore
Feature: re-usable feature to create a single user

  Scenario:

    * def utils = callonce read('classpath:utils.feature')

    * def username = utils.randomEmail()
    * def password = utils.randomPassword()
    * def stripeToken = utils.randomStripeToken()
    * def first = utils.randomAlphabetic(5)
    * def last = utils.randomAlphabetic(8)
    * def address1 = utils.randomAlphabetic(15)
    * def city = utils.randomAlphabetic(9)
    * def state = 'FL'
    * def zip = 12345
    * def phone = utils.randomNumeric(10)
    * def testPriceTierName = (typeof priceTierName === 'undefined') ? 'BASIC' : priceTierName
    * def testTrialSubscription = (typeof trialSubscription === 'undefined') ? true : trialSubscription
    * def source = 'karate'

    * def hideLogRegister = (typeof hideLog === 'undefined') ? true : hideLog

    * def userRegistration = read('this:user.json')

    * if (typeof cardToken !== 'undefined') userRegistration.cardToken = cardToken
    * if (typeof couponCode !== 'undefined') userRegistration.couponCode = couponCode
    * if (typeof additionalProperties !== 'undefined') userRegistration.additionalProperties = additionalProperties

    * def originalLogLevelRegister = hideLogRegister ? utils.getLogLevel() : false
    * if (hideLogRegister) utils.setLogLevel('ERROR')

  #    * if (localstack) originalMessageCount = call read('check-localstack-emails.feature')
    * def originalMessageCount = localstack ? karate.call('this:../utils/check-localstack-emails.feature') : 0
```

**Comment by ptrthomas** [comment]
@arnaldop thank you very much for reporting this - it was an edge case for a JS line followed by a comment

this is not yet on the marketplace, but we uploaded a version so you could validate it is fixed: 2.0.4.37 - https://github.com/karatelabs/intellij-plugin/releases/tag/v2.0.0.EAP

**Comment by ptrthomas** [comment]
@arnaldop 2.0.4 from the marketplace is now availablle, the published version includes support for `switch case` so please un-install your existing version if needed
